lib/v2/components/text_form_field_custom.dart

```diff
@@ -34,10 +34,10 @@ class TextFormFieldCustom extends StatelessWidget {
     this.keyboardType = TextInputType.text,
     this.controller,
     this.onChanged,
-    this.textCapitalization,
+    this.textCapitalization = TextCapitalization.none,
     this.inputFormatters,
     this.maxLength,
-    this.maxLines,
+    this.maxLines = 1,
```

Comment:
what if we need this to be more than 1 line?

Reply:
I only extend the default value.
![image](https://user-images.githubusercontent.com/42857405/109194499-1605a800-775f-11eb-9139-3f183371be7c.png)